Composer/packages/adaptive-flow/src/adaptive-flow-editor/renderers/EdgeMenu/createSchemaMenu.tsx

```diff
@@ -174,13 +176,48 @@ interface ActionMenuOptions {
 const createSubMenu = (
   label: string,
   onClick: ActionMenuItemClickHandler,
-  subItems: IContextualMenuItem[]
+  subItems: IContextualMenuItem[],
+  forceDisabledActions: DisabledMenuActions[]
 ): IContextualMenuItem => {
+  const subMenuItems = subItems.map((subMenuItem: IContextualMenuItem) => {
+    let additionalProps: Partial<IContextualMenuItem> = {};
+    const disabledAction = forceDisabledActions.find((action) => action.kind === subMenuItem.key);
+    if (disabledAction) {
+      additionalProps = {
+        title: disabledAction.reason,
+        disabled: true,
+        onRender: (item) => {
+          const calloutProps = { gapSpace: 0 };
+          const tooltipId = `tooltip-${disabledAction.kind}`;
+          const hostStyles: Partial<ITooltipHostStyles> = {
+            root: {
+              display: 'flex',
+              justifyContent: 'flex-start',
+              alignItems: 'center',
+              padding: '0px 7px',
+              height: '38px',
+              background: '#F3F2F1',
```

Comment:
use NeutralGray equivalent of this color, helps in the future with theming (Gray20)
https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/neutrals